modules/cli/pages/cli-intro.adoc

```diff
@@ -1,70 +1,82 @@
 = CLI Reference
-:description: Couchbase Server command-line interface (CLI) tools are provided to manage and monitor clusters, servers, vBuckets, XDCR, and so on.
+:description: The command-line interface (CLI) tools lets you manage and monitor your Couchbase Server installation including clusters, servers, vBuckets, and XDCR.
+:tools-ver: 7.6.2
 
 [abstract]
 {description}
 
-By default, the CLI tools are installed into the following locations on each platform:
+The Couchbase Server installation process installs the command-line tools.
+The location of these tools depends on your platform:
```

Comment:
Maybe make it clear that this is where you can find the tools after installation?

```suggestion
When you have installed Couchbase Server, you can find the command-line tools in one of the following locations, depending on your your platform:
```